presentation/placeholder-add-table/main.go

```diff
@@ -0,0 +1,83 @@
+// Copyright UniDoc ehf. All rights reserved.
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/unidoc/unioffice/common/license"
+	"github.com/unidoc/unioffice/measurement"
+	"github.com/unidoc/unioffice/presentation"
+	"github.com/unidoc/unioffice/schema/soo/dml"
+	"github.com/unidoc/unioffice/schema/soo/pml"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	ppt, err := presentation.Open("placeholder.pptx")
+	if err != nil {
+		panic(err)
```

Comment:
Maybe log some error, not panic?

Reply:
updated at 8379fee68c74ec161e445c4d6fad986a6d8be406